charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml

```diff
@@ -24,8 +24,8 @@ spec:
 {{ toYaml .Values.labels | indent 8 }}
         networking.gardener.cloud/to-dns: allowed
         networking.gardener.cloud/to-runtime-apiserver: allowed
-        networking.resources.gardener.cloud/to-logging-tcp-metrics: allowed
-        networking.resources.gardener.cloud/to-all-shoots-loki-tcp-metrics: allowed
+        networking.resources.gardener.cloud/to-logging-tcp-3100: allowed
```

Comment:
About https://github.com/cilium/cilium/issues/24779? Duplicate port with another Pod's port in the same namespace? If that's the case don't we have almost every component with such a metrics port?